documentation/src/main/asciidoc/topics/code_examples/AnalyzerDefinitions.java

```diff
@@ -0,0 +1,7 @@
+@ProtoDoc("@Field(store = Store.YES, analyze = Analyze.YES, analyzer = @Analyzer(definition = \"keyword\"))")
+@ProtoField(number = 1)
```

Comment:
is number mandatory? I think we can add just 1 instead of putting number (even if I have been putting number everywhere lately)

Reply:
@gustavonalle Can you please take a look at this one?

Reply:
```@ProtoField(1)``` is valid